Header header

momav/Remote     remote				# timeout flag for remote topic
momav/BodyState  body_state			# timeout flag for body state topic (body_state_track, body_state_cam or body_state_sim)
momav/BodySetp   body_setp_pose		# timeout flag for body pose setpoint topic
momav/BodySetp   body_setp_wrench	# timeout flag for body wrench setpoint topic
momav/MotorSetp  motor_setp			# timeout flag for motor setpoints topic
momav/MotorState motor_state		# timeout flag for motor states topic
momav/ServoSetp  servo_setp			# timeout flag for servo setpoints topic
momav/ServoState servo_state		# timeout flag for servo states topic
